taxi_service/settings.py

```diff
@@ -53,10 +54,12 @@
 
 ROOT_URLCONF = "taxi_service.urls"
 
+BASE_DIR = Path(__file__).resolve().parent.parent
+
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": [os.path.join(BASE_DIR, "taxi/../templates")],
```

Comment:
Code quality issue: It looks like you're trying to join the BASE_DIR with a relative path to the templates directory. However, the way it's written now, it could lead to unexpected results. Use os.path.join(BASE_DIR, 'templates') instead.